libs/docs-utils/src/api/role.enum.ts

```diff
@@ -0,0 +1,21 @@
+export enum DaffDocsApiRole {
```

Comment:
Will these be used anywhere else other than to display the headings of each role? Because the headings should be pluralized, but they may not have to be if used somewhere else.

Reply:
These values should not be directly displayed. I simply have them here as an example. I'll add a transform to capitalize them, is that sufficient? Do you have a specific order in which you want the sections rendered?

Reply:
They should be sentence cased and pluralized. i.e. Components, Directives, etc.

I think it should be Components, Directives, Services, and then I don't have an opinion on the rest. What do you think?

Reply:
fixed

Reply:
> They should be sentence cased and pluralized. i.e. Components, Directives, etc.
> 
> I think it should be Components, Directives, Services, and then I don't have an opinion on the rest. What do you think?

I don't have a strong opinion honestly